packages/salesforcedx-vscode-core/src/commands/forceSourceStatus.ts

```diff
@@ -41,14 +45,46 @@ export const statusCommand: CommandParams = {
 
 export const statusCommandLegacy: CommandParams = {
   command: 'force:source:legacy:status',
-  description: {default: 'force_source_legacy_status_text'},
-  logName: {default: 'force_source_legacy_status'}
+  description: { default: 'force_source_legacy_status_text' },
+  logName: { default: 'force_source_legacy_status' }
 };
 
+export class SourceTrackingGetStatusExecutor extends LibraryCommandletExecutor<
+  string
+> {
+  private flag: SourceStatusFlags | undefined;
+
+  constructor(
+    executionName: string,
+    logName: string,
+    flag?: SourceStatusFlags | undefined
+  ) {
+    super(nls.localize(executionName), logName, OUTPUT_CHANNEL);
+    this.flag = flag;
+  }
+
+  public async run(response: ContinueResponse<string>): Promise<boolean> {
+    const trackingService = new SourceTrackingService();
+    const sourceStatusOptions = {
+      remote: !(this.flag && this.flag === SourceStatusFlags.Local)
+    };
+    const sourceStatusSummary: string = await trackingService.getSourceStatusSummary(
+      sourceStatusOptions
+    );
+    channelService.appendLine('Source Status');
```

Comment:
Should 'Source Status' be localized?

Reply:
Since it is going to be viewed by the end user, I think it should be localied.